istioctl/pkg/writer/pilot/status.go

```diff
@@ -176,8 +178,13 @@ func (s *XdsStatusWriter) setupStatusPrint(drs map[string]*xdsapi.DiscoveryRespo
 				}
 				cds, lds, eds, rds := getSyncStatus(&clientConfig)
 				cp := multixds.CpInfo(dr)
+				meta, err := model.ParseMetadata(clientConfig.GetNode().GetMetadata())
```

Comment:
Is it possible to get the clusterID via similar way from `getSyncStatus` , just like get `cds, lds, eds, rds` above?

Reply:
Those sync statues are generated from the `GenericXdsConfigs` in `clientConfig`, and `clusterID, proxyID` are from the `Node` info in the `clientConfig`. I think they should not be mixed up, but there could be an improvement in the next to unify them to a function like `getProxyStatuses`.

Reply:
@zhlsunshine PTAL, thanks!